.mailmap

```diff
@@ -567,7 +568,6 @@ Aman Sharma <amansharma110603@gmail.com> mostlyaman <85072876+mostlyaman@users.n
 Robin Richard <raisin@ecomail.fr> robinechuca <raisin@ecomail.fr>
 Robin Richard <raisin@ecomail.fr> robinechuca <61911191+robinechuca@users.noreply.github.com>
 Robin Richard <raisin@ecomail.fr> Nornort <francois.lalleve@laposte.net>
-Ayush Kumar <ayushk7102@gmail.com> Ayush Kumar <65803868+ayushk7102@users.noreply.github.com>
```

Comment:
Why is this deleted?

Reply:
I'm not sure but i guess that was done after rebasing.

Reply:
I guess it is not needed because the 65803868 email address does not seem to be in the git log.

Reply:
I checked the commit log and it seems like there is no "GitHub account" associated with some recent commits by Ayush Kumar, i.e., it is not possible to click the name for his commits. Probably has nothing to do with this, but since it is the same person I thought it was worth mentioning here.